pkg/compliant/dcr32_config.go

```diff
@@ -81,7 +87,9 @@ func NewDCR32Config(
 		WithPrivateKey(privateKey).
 		WithTokenEndpointAuthMethod(tokenSignMethod).
 		WithTransportCert(transportCert).
-		WithTransportCertSubjectDn(transportCertSubjectDn)
+		WithTransportCertSubjectDn(transportCertSubjectDn).
+		WithSSAs(ssas).
+		WithSSAsPresence(SSAsPresents)
```

Comment:
Please explain why this method is necessary.

Reply:
This method is used to detect if multiple or single SSA was set in the config file. The information is stored under ssasPresence field and it is used to decide if SSAs mechanism is required.